src/flow_solver/flow_solver.cpp

```diff
@@ -441,8 +441,13 @@ int FlowSolver<dim,nstate>::run() const
         //----------------------------------------------------
         double time_step = 0.0;
         if(flow_solver_param.adaptive_time_step == true) {
+            pcout << "WARNING: adaptive time step is different from error adaptive time step " << std::endl;
```

Comment:
I think it might be best to abort as suggested. From the code it looks like the test will move forward with the adaptive time step instead of the error adaptive step and if the user misses the warning then the results could be confusing since a constant time step would be used throughout.

Reply:
Ok thanks this is a good idea! I implemented this